ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1036,3 +1036,36 @@ def test_tensorflow_truediv(
         x=x[0],
         y=x[1],
     )
+    
+  
+# argmin
+@handle_cmd_line_args
+@given(
+    dtype_and_x=statistical_dtype_values(function="argmin"),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.math.argmin"
+    ), output_types=helpers.get_dtypes("numeric"),
+)
+def test_tensorflow_argmin(
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+    output_types,
+):
+    input_dtype, x, axis = dtype_and_x
```

Comment:
Since output_type can only be int32 or int64, we should generate it as:
`output_type = random.choice(["int64", "int32"])`

Reply:
I see in your commits that you were trying to do that in the test function's arguments. That's not what I meant. I meant use the `random.choice` in the function's body, after `input_dtype, x, axis = dtype_and_x`.